### PR TITLE
Fureeous ashenvale horde

### PR DESCRIPTION
**Pull request by mdavidson37**
Minor fixes:

- Gorat's Corpse: removed erroneous |FLY|OLD| Tag
- Small hands, Short Fuse: Blizz has renamed to Small Stature, Short Fuse
- Tweedle's Tiny Package: Blizz has renamed to Tweedle's Tiny Parcel
- The Essence of Aku'Mai: Changed to rank 1 as it is required for Loremaster achieve
- Silverwind Refuge: Fixed step to set hearth.

### CODE REVIEW

File: WoWPro_Leveling/Horde/CATA_Ashenvale.lua [hunk 1]
```diff
@@ -146,7 +146,7 @@ T Bad News Bear-er|QID|13848|M|12.10,33.85|N|Turn in to Commander Grimfang.|
 A Keep the Fires Burning|QID|13890|M|12.10,33.85|N|From Commander Grimfang.|RANK|1|
 A Blackfathom Deeps|QID|26894|M|12.10,33.85|ELITE|N|[color=E6CC80]Dungeon: Blackfathom Deeps[/color]\nFrom Commander Grimfang.\nPick this up if you plan on running the Blackfathom Deeps instance.|RANK|3|
 A Lousy Pieces of Ship|QID|13883|M|11.57,35.31|N|From Dagrun Ragehammer. He paces back and forth.|RANK|2|
-A The Essence of Aku'Mai|QID|26890|M|11.57,35.31|ELITE|N|[color=E6CC80]Dungeon: Blackfathom Deeps[/color]\nFrom Dagrun Ragehammer.|RANK|3|
+A The Essence of Aku'Mai|QID|26890|M|11.57,35.31|ELITE|N|From Dagrun Ragehammer.|RANK|2|
```
Comment (Hendo72) [inline WoWPro_Leveling/Horde/CATA_Ashenvale.lua]: Remove the ELITE tag. Not required anymore. Secondly, it should be RANK|1| not 2.

File: WoWPro_Leveling/Horde/CATA_Ashenvale.lua [hunk 2]
```diff
@@ -217,10 +217,10 @@ T Freedom to Ruul|QID|6482|M|74.11,60.91|N|To Yama Snowhoof.|
 K Sharptalon|AVAILABLE|2|M|72.33,76.83|L|16305|N|Go kill Sharptalon and loot the claw.\n[color=FF0000]NOTE: [/color]You are leaving the area and this is the last opportunity to do this.|T|Sharptalon|US|RANK|3|
 F Silverwind Refuge|ACTIVE|13974|M|38.09,42.23|N|Fly to Silverwind Refuge.|TAXI|Silverwind Refuge|
 A Thinning the... Herd?|QID|13967|M|49.75,65.08|N|From Senani Thunderheart.|RANK|2|
-T Tweedle's Tiny Package|QID|13974|M|49.96,67.25|N|To Flooz.|
+T Tweedle's Tiny Parcel|QID|13974|M|49.96,67.25|N|To Flooz.|
 A Mass Production|QID|13977|M|49.96,67.25|N|Wait a moment then get this quest from Flooz.|PRE|13974|RANK|2|
 A Simmer Down Now|QID|25|M|50.13,67.58|N|From Captain Tarkan.|RANK|2|
-h Silverwind Refuge|ACTIVE|6482|M|50.43,67.16|N|At Innkeeper Chin'toka.|
+h Silverwind Refuge|ACTIVE|25|M|50.43,67.16|N|At Innkeeper Chin'toka.|
```
Comment (Hendo72) [inline WoWPro_Leveling/Horde/CATA_Ashenvale.lua:223]: I would change this to |AVAILABLE|26416|. Prevents surviving resets.

**Review by Hendo72 — APPROVED** [review]

**Comment by mdavidson37** [comment]
> Does the following C step not accomplish the same thing? I suggest deleting the R step and adding to the note to run to the corpse

I felt that it didn't. The quest flow is that you need to go to C and revive Gorat - who then walks to the R step location and summons Captain Elendilad for you to kill.

If you go straight to the C step there's nothing there and they are far apart enough for it to be non-obvious where to go unless you look at the quest marker. Arguably you could remove the R step and add a note to the C step to follow Gorat to kill the Captain.

**Review by Cagomei — CHANGES_REQUESTED** [review]
From testing these changes removing the FLY and OLD are needed. i would suggest leaving the R, and changing the C to a K with a |QO|1| for the visual of this is a named mob that needs to be killed.

**Review by Cagomei — APPROVED** [review]

**Comment by Ludovicus-Maior** [comment]
@mdavidson37 Sorry for the grilling.

**Pull request merged**